rollouts/controllers/remoterootsync_controller.go

```diff
@@ -125,6 +126,8 @@ func (r *RemoteRootSyncReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 				// Delete the external sync resource
 				err := r.deleteExternalResources(ctx, &remoterootsync)
 				if err != nil && !apierrors.IsNotFound(err) {
+					r.updateStatus(ctx, &remoterootsync, "", err)
```

Comment:
Maybe log the error here if this fails. Not much else we can do at this point, but it would at least give some hint somewhere that something failed.

Reply:
Can do. Will push a commit.